app/src/main/python/gogdl/dl/managers/manager.py

```
@@ -0,0 +1,202 @@
+            self.builds = self.get_builds(self.platform)
+        except UnsupportedPlatform:
+            if self.platform == "linux":
+                self.logger.info(
+                    "Platform is Linux, redirecting download to Linux Native installer manager"
+                )
+
+                self.download_manager = linux.Manager(self)
+
+                return
+
+            self.logger.error(f"Game doesn't support content system api, unable to proceed using platform {self.platform}")
```

Ruff (0.14.8)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)